.github/workflows/publish.yaml

```diff
@@ -8,25 +8,8 @@ permissions:
   contents: read
 
 jobs:
-  deploy:
-    environment: release
+  noop:
     runs-on: ubuntu-latest
-
     steps:
-      - uses: actions/checkout@v4
-        with:
-          persist-credentials: false
-      - name: Set up Python
-        uses: actions/setup-python@v5
-        with:
-          python-version: "3.11"
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          pip install hatch
-      - name: Build package
-        run: sh ${{ github.workspace }}/.compat/build_snowflake-cli-labs.sh
-      - name: Publish package
-        run: |
-          cd ${{ github.workspace }}./compat/snowflake-cli-labs/
-          python -m hatch publish -u __token__ -a ${{ secrets.PYPI_TOKEN }}
+      - name: Create release tag
+        run: echo "Release created for ${{ github.event.release.tag_name }}"
```

Comment:
Do we need this job?

Reply:
To be honest not really.
I removed it completely.